tests/ast/test_pre_parser.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from bumpversion.version_part import VersionConfig
+from vyper import __version__
+from vyper.ast.pre_parser import validate_version_pragma
+from vyper.exceptions import VersionException
+
+SRC_LINE = (1, 0)
```

Comment:
Just add a comment that this is a dummy source line:
```suggestion
SRC_LINE = (1, 0)  # Dummy source line
```

Reply:
Aye!